sys/net/gnrc/application_layer/dhcpv6/client_simple_pd.c

```diff
@@ -101,8 +101,13 @@ static void _configure_upstream_netif(gnrc_netif_t *upstream_netif)
 static void _configure_dhcpv6_client(void)
 {
     gnrc_netif_t *netif = NULL;
+    gnrc_netif_t *upstream = _find_upstream_netif();
     while ((netif = gnrc_netif_iter(netif))) {
-        if (gnrc_netif_is_6lo(netif)) {
+        if (IS_USED(MODULE_GNRC_DHCPV6_CLIENT_6LBR)
+            && !gnrc_netif_is_6lo(netif)) {
+            continue;
+        }
+        if (netif != upstream) {
             dhcpv6_client_req_ia_pd(netif->pid, 64U);
         }
```

Comment:
I don't see why we would want that, now if only `gnrc_dhcpv6_client_simple_pd` is selected this is a bug as also the upstream interface is selected for prefix delegation.

I think there is no need for a 'greedy' module: `gnrc_dhcpv6_client_simple_pd` could just be greedy by default if not specified otherwise (e.g. by excluding non-6lo interfaces)

Reply:
Ok